src/site/content/en/blog/ar-hit-test/index.md

```diff
@@ -289,7 +289,7 @@ reticle gives you a constant source of hit tests, the simplest way to place an
 object is to draw it at the location of the reticle at the last hit test. If you
 need to, say you have a legitimate reason not to show a reticle, you can call
 `requestHitTest()` in the select event [as shown in the
-sample](https://github.com/immersive-web/webxr-samples/blob/master/proposals/phone-ar-hit-test.html#L187).
+sample](https://github.com/immersive-web/webxr-samples/blob/429aeb7cd46f1009d2e529e69854418a5a0903d5/proposals/phone-ar-hit-test.html#L189).
```

Comment:
@toji What's the updated location for the Phone AR Hit Test? We should probably do something better than linking to a commit blob.

Reply:
Live Page: https://immersive-web.github.io/webxr-samples/hit-test.html
Source: https://github.com/immersive-web/webxr-samples/blob/master/hit-test.html

Reply:
@beaufortfrancois Please use Brandon's links.

Reply:
@jpmedley I've linked to a commit blob because the article references `requestHitTest` which doesn't exist in the updated live page/source pages shared by @toji.
Unless you're willing to update the article, we should use those links.

Reply:
@toji It's called `requestHitTestSource()` now, right?

Reply:
Correct, used here: https://github.com/immersive-web/webxr-samples/blob/master/hit-test.html#L137

But this does bring up a good point, in that you'll need to update the surrounding text as well because the method isn't called in the select event any more. The equivalent call in frame callbacks or select events is now `frame.getHitTestResults()` https://github.com/immersive-web/webxr-samples/blob/master/hit-test.html#L199

Reply:
I opened #2394 to address these links and surrounding text for real.